bin/generate

```diff
@@ -39,11 +43,11 @@ func curryDefinitionGenerator(arguments: Int) -> String {
   let lowerFunctionArguments = inputParameters.map { "\($0.lowercased())" } // ["a", "b", "c"]
   let returnType = genericParameters.last! // "D"
 
-  let functionArguments = "(\(commaConcat(inputParameters)))" // "(A, B, C)"
+  let functionArguments = extraParentheses("(\(commaConcat(inputParameters)))", when: inputParameters.count > 1) // "(A, B, C)"
```

Comment:
what happens if there is only one argument? does it error out? could we skip this condition?